utils/hare-node

```diff
@@ -17,9 +17,39 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
-set -eu -o pipefail
-# set -x
-export PS4='+ [${BASH_SOURCE[0]##*/}:${LINENO}${FUNCNAME[0]:+:${FUNCNAME[0]}}] '
+# :help: stop the cluster
 
-# :help: manage the cluster nodes
-PCSCLI_PROG_NAME='hctl node' pcswrap "$@"
+import getopt
+import sys
+from socket import gethostname
+from util import setup_logging, stop_parallel
+from util import processes_node, shutdown_sequence
+from consul import Consul
+
+
+def local_node_shutdown():
+
```

Comment:
delete this extra line please.

Reply:
@gshipra There are linters for Python that can make the code prettier just automatically. For instance, I use [`yapf`](https://github.com/google/yapf) and [`isort`](https://pypi.org/project/isort/). 

If you use vim, there is an easy way to integrate them into the editor.

Reply:
@knekrasov - Thanks for letting me know. I will try these linters.

Reply:
deleted extra line.